tests/test_context/test_requires_context/test_context.py

```diff
@@ -21,6 +23,24 @@ def test_protocols(container, protocol):
 
 
 def test_context_immutable():
-    """Ensures that RequiresContext container supports ``.map()`` method."""
+    """Ensures that Context is immutable."""
     with pytest.raises(ImmutableStateError):
         Context().abc = 1
+
+
+def test_context_immutable_copy():
+    """  # noqa
```

Comment:
Can you please refactor this doctest to take a single line and remove `noqa`?
I think that there's no need for it here.